src/layouts/Meta.tsx

```diff
@@ -13,16 +14,21 @@ type IMetaProps = {
 
 const Meta = (props: IMetaProps) => {
   const router = useRouter();
+  const [viewport, setViewport] = useState(
+    'width=device-width, initial-scale=1',
+  );
+
+  useEffect(() => {
+    if (navigator.userAgent.indexOf('iPhone') > -1) {
+      setViewport('width=device-width, initial-scale=1, maximum-scale=1');
+    }
+  }, []);
```

Comment:
what's this, and why have we added it?

Reply:
IOS Safari Browser auto zooms in on the input boxes when focused.

this code block helps to disable that auto-zoom

Reply:
but why inside a useEffect??
why not just change the meta viewport instead?

Reply:
coz we want to apply it just for iphone,
if u set maximum-scale=1 on android, android users wont be able to zoom in

Reply:
https://weblog.west-wind.com/posts/2023/Apr/17/Preventing-iOS-Safari-Textbox-Zooming#selective-maximum-scale-for-safari-on-ios